verification/curator-service/api/src/util/validate-env.ts

```diff
@@ -16,7 +16,7 @@ export default function validateEnv(): Readonly<{
         }),
         DB_CONNECTION_STRING: str({
             desc: 'MongoDB URI provided to MongoClient.',
-            devDefault: 'mongodb://localhost:27017/dev',
```

Comment:
There's probably an analogous default to update in data-service.

Reply:
good catch, done.